virttest/qemu_devices/qcontainer.py

```diff
@@ -1395,6 +1395,9 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
         devices[-1].set_param('snapshot', snapshot, bool)
         devices[-1].set_param('readonly', readonly, bool)
         if 'aio' in self.get_help_text():
+            if aio == 'native' and snapshot == 'on':
```

Comment:
Hi xu,
i apply the patch and try the case , find that it didn't works ,so i check the cfg and find this variable "image_snapshot = yes",so i think it should be snapshot == 'yes', thanks.